### ISSUE TITLE
Static assets are copied into root folder

### ISSUE THREAD

**Issue by andreasmcdermott** [antony/sveltekit-adapter-browser-extension#2]
I have a Svelte project that looks like this:

```
- src
  - lib
  - routes
  - app.html
  - ...
- static
  - icon.png
  - logo.png 
  - ...
```

When building, the adapter puts images in `static` directly into `build` instead of `build/static` which is what I'd expect. Currently I modified the adapter like so to work around it:

```
// Before:
utils.copy_static_files(assets);

// After:
utils.copy_static_files(`${assets}/static`);
```

**Comment by antony** [comment]
@andreasmcdermott that's right - it acts much more like a regular adapter in this respect - so that files appear at the root `/` of your site, the `static` prefix is dropped.

I don't see why this has to be a constraint though, so maybe it can change.
